src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -177,6 +180,12 @@ export const ContentMessageComponent = ({
               {timeAgo}
             </MessageTime>
           </span>
+          <ReadIndicator
+            message={lastMessageInGroup}
+            is1to1Conversation={conversation.is1to1()}
+            isLastDeliveredMessage={isLastDeliveredMessage || lastMessageInGroup.status() === StatusType.DELIVERED}
+            showIconOnly
+          />
```

Comment:
Instead of the message directly after the header, we pass the last message in the group to the read indicator.